### PR TITLE
change a couple of metatype attribute names

### PR DESCRIPTION
**Pull request by brutif**

### CODE REVIEW

**Comment by brutif** [comment]
#build

**Comment by brutif** [comment]
#build

File: dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -20,20 +20,21 @@ jwtSso.desc=Information about configuring JWT Single Sign On.
...
 
 includeLtpaCookie=Include LTPA cookie
-includeLtpaCookie.desc=After successful authentication with a JWT token, include an LTPA cookie in addition to the SSO cookie.
+includeLtpaCookie.desc=After successful authentication with a JWT token, include an LTPA cookie in addition to the JWT cookie.
 
-fallbackToLtpa=Fall back to LTPA cookie
-fallbackToLtpa.desc=If the SSO cookie is missing or invalid, attempt to process an LTPA cookie.
+#initially this property was exposed as fallbackToLtpa but now is useLtpaIfJwtAbsent
+fallbackToLtpa=Use LTPA cookie if JWT cookie is not present
+fallbackToLtpa.desc=If the JWT cookie is missing or invalid, attempt to process an LTPA cookie if present.
```
Comment (ayoho) [inline dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties]: I thought we'll only try to process the LTPA cookie if the JWT cookie is missing; if the JWT cookie is invalid, it was my understanding that we would immediately fail and return a 401 without falling back to LTPA.

**Comment by brutif** [comment]
#build

File: dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (brutif) [reply dev/com.ibm.ws.security.jwtsso/resources/OSGI-INF/l10n/metatype.properties]: yep, you're right, Ut has updated SsoAuthenticator.  I'll update the props file.

**Comment by ayoho** [comment]
#build

**Comment by brutif** [comment]
build died with out of memory problems, let's try again
#build

**Pull request merged**